src/components/atoms/TextInput/TextInput.tsx

```diff
@@ -1,29 +1,40 @@
+import { useState } from 'react';
+
+interface TextInputProps {
+  id?: string;
+  type?: string;
+  placeholder?: string;
```

Comment:
are these not always defined?

Reply:
Nope, default values are used in Home.tsx